mobile/fastlane/Fastfile

```diff
@@ -0,0 +1,224 @@
+# This file defines the build and signing lanes for iOS
+
+default_platform(:ios)
+
+TEAM_ID = "8B5X2M6H2Y"
+APP_ID_RELEASE = "app.status.mobile"
+APP_ID_PR = "app.status.mobile.pr"
+APP_NAME_RELEASE = "Status.app"
+APP_NAME_PR = "StatusPR.app"
+PROJECT_DIR = File.expand_path("../", __dir__)
+BUILD_DIR = File.join(PROJECT_DIR, "bin", "ios", "qt6")
+
+platform :ios do
+  before_all do
+    UI.message("Project directory: #{PROJECT_DIR}")
+    UI.message("Build directory: #{BUILD_DIR}")
+  end
+
+  after_all do
+    # Clean up CI keychain after build
+    if is_ci
+      delete_keychain(name: keychain_name) if File.exist?(File.expand_path("~/Library/Keychains/#{keychain_name}-db"))
+    end
+  end
+
+  error do
+    # Clean up CI keychain on failure too
+    if is_ci
+      delete_keychain(name: keychain_name) rescue nil
+    end
+  end
+
+  # ============================================
+  # PR Builds
+  # ============================================
+  desc "Build iOS app for PRs"
+  lane :pr do
+    setup_ci_keychain
+
+    run_match(type: "adhoc", app_identifier: APP_ID_PR)
+
+    sign_app(
+      app_identifier: APP_ID_PR,
+      app_name: APP_NAME_PR,
+      profile_type: "adhoc"
+    )
+
+    create_ipa(app_name: APP_NAME_PR)
+  end
+
+  # ============================================
+  # Release Builds
+  # ============================================
+  desc "Build iOS app for release"
+  lane :release do
+    setup_ci_keychain
+
+    run_match(type: "appstore", app_identifier: APP_ID_RELEASE)
+
+    sign_app(
+      app_identifier: APP_ID_RELEASE,
+      app_name: APP_NAME_RELEASE,
+      profile_type: "appstore"
+    )
+
+    create_ipa(app_name: APP_NAME_RELEASE)
+
+    if ENV["UPLOAD_TO_TESTFLIGHT"] == "true"
+      upload_to_testflight_lane
```

Comment:
Why dont we use it? Why `upload-testflight.sh`?

Reply:
I think we need only one and I would prefer action from Fastlane compared to homebuilt script.